diff --git a/dist/shared/nitro.c8278d90.mjs b/dist/shared/nitro.c8278d90.mjs
index 9ba312fc248da3731720ee7e3b38ba2a85537657..5ec9f06ccf60259820586715d73d41466daa8cff 100644
--- a/dist/shared/nitro.c8278d90.mjs
+++ b/dist/shared/nitro.c8278d90.mjs
@@ -1296,7 +1296,7 @@ async function copyPublicAssets(nitro) {
   if (nitro.options.noPublicDir) {
     return;
   }
-  for (const asset of nitro.options.publicAssets) {
+  for (const asset of [...nitro.options.publicAssets].reverse()) {
     if (await isDirectory(asset.dir)) {
       await fse.copy(asset.dir, join(nitro.options.output.publicDir, asset.baseURL));
     }