diff --git a/dist/module.d.ts b/dist/module.d.ts
index 02943d62e7c28d356daca8ac969e75584014cf7b..a2074ea0dad2804354732b6901c25eefdc831a30 100644
--- a/dist/module.d.ts
+++ b/dist/module.d.ts
@@ -221,15 +221,15 @@ interface NuxtSecurityRouteRules {
 
 declare module "@nuxt/schema" {
     interface NuxtOptions {
-        security: ModuleOptions;
+        security?: ModuleOptions;
     }
 }
 declare module "nitropack" {
     interface NitroRouteRules {
-        security: NuxtSecurityRouteRules;
+        security?: NuxtSecurityRouteRules;
     }
     interface NitroRouteConfig {
-        security: NuxtSecurityRouteRules;
+        security?: NuxtSecurityRouteRules;
     }
 }
 declare const _default: _nuxt_schema.NuxtModule<ModuleOptions>;