diff --git a/dist/module.d.ts b/dist/module.d.ts
index b529dffc651fdd47ccfcf19b896cf76bbe68a0a6..3db4fe583892e6f092dc09f4f11c1a30606d1b20 100644
--- a/dist/module.d.ts
+++ b/dist/module.d.ts
@@ -232,15 +232,15 @@ interface NuxtSecurityRouteRules {
 
 declare module "@nuxt/schema" {
     interface NuxtOptions {
-        security: ModuleOptions;
+        security?: ModuleOptions;
     }
 }
 declare module "nitropack" {
     interface NitroRouteRules {
-        security: NuxtSecurityRouteRules;
+        security?: NuxtSecurityRouteRules;
     }
     interface NitroRouteConfig {
-        security: NuxtSecurityRouteRules;
+        security?: NuxtSecurityRouteRules;
     }
 }
 declare const _default: _nuxt_schema.NuxtModule<ModuleOptions>;
